lambdas/send_from_queue/send_from_queue.py

```diff
@@ -208,19 +208,34 @@ def lambda_handler(event, _context):
                     "queue. {}".format(message_body)
                 )
 
+            api_url_full_query = api_url
+            already_triggered = message_body.get("already_triggered")
+            if already_triggered:
+                    api_url_full_query = api_url_full_query + \
+                        "&already_triggered={}".format(already_triggered)
+
```

Comment:
No, its saying if the hook contains anything for already_triggered, include it in the API call so the lambda know what has been previously executed.